ftw/portlet/weather/profiles/default/metadata.xml

```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0"?>
 <metadata>
-  <version>1</version>
+  <dependencies>
+    <dependency>profile-ftw.upgrade:default</dependency>
```

Comment:
This only installs the upgrade control panel and is not necessary here.
The Portlet does not require the control panel action to be there, it just requires the upgrade step director feature.
This can be done in policy packages...